openff/toolkit/topology/molecule.py

```diff
@@ -5999,7 +6013,7 @@ def add_bond_charge_virtual_site(self, atoms, distance, **kwargs):
 
         Create a bond charge-type virtual site, in which the location of the
         charge is specified by the position of two atoms. This supports
-        placement of a virtual site S along a vector between two specified
+        placement of a virtual site $S$ along a vector between two specified
```

Comment:
This `$S$` renders verbatim as written

```suggestion
        placement of a virtual site $S$ along a vector between two specified
```